img/t/tripadvisor.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 3354 714"><circle cx="356.7" cy="356.7" r="356.7" fill="#34e0a1"/><path d="m577 287 43-47h-95a296 296 0 0 0-336 0H93l43 47a131 131 0 1 0 179 194l42 46 42-46a132 132 0 1 0 178-194zM225 473a89 89 0 1 1 0-178 89 89 0 0 1 0 178zm132-91c0-59-43-109-99-131a256 256 0 0 1 198 0c-57 22-99 72-99 131zm131 91a89 89 0 1 1 0-178 89 89 0 0 1 0 178zm0-135a47 47 0 1 0 0 93 47 47 0 0 0 0-93zm-216 46a47 47 0 1 1-93 0 47 47 0 0 1 93 0z"/></svg>
```

Comment:
I find two issues here.
1. The viewBox is incorrect, causing the logo to be pushed to a small part of the image.
  ![screenshot](https://user-images.githubusercontent.com/3535780/202298564-c957e9a4-53ba-4408-a66e-2624eb34ad03.png)
2. You changed the number precision too which caused parts of the logo to become distorted. See the right eye of the owl :laughing:
  ![](https://user-images.githubusercontent.com/3535780/202298837-b8529af4-9d52-4fc3-b7d0-24de40909c65.png)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 713.496 713.496"><circle cx="356.748" cy="356.748" r="356.748" fill="#34e0a1"/><path d="m577.094 287.152 43.049-46.836h-95.465c-47.792-32.646-105.51-51.659-167.931-51.659-62.342 0-119.899 19.054-167.612 51.659H93.431l43.049 46.836c-26.387 24.075-42.929 58.754-42.929 97.259 0 72.665 58.914 131.578 131.579 131.578 34.519 0 65.968-13.313 89.446-35.077l42.172 45.919 42.172-45.879c23.478 21.764 54.887 35.037 89.406 35.037 72.665 0 131.658-58.913 131.658-131.578.039-38.545-16.502-73.223-42.89-97.259zM225.169 473.458c-49.188 0-89.047-39.859-89.047-89.047 0-49.188 39.86-89.048 89.047-89.048s89.047 39.86 89.047 89.048-39.86 89.047-89.047 89.047zm131.618-91.638c0-58.595-42.61-108.898-98.853-130.383 30.413-12.716 63.776-19.771 98.813-19.771s68.439 7.055 98.853 19.771c-56.202 21.525-98.813 71.789-98.813 130.383zm131.579 91.638c-49.188 0-89.048-39.859-89.048-89.047 0-49.188 39.86-89.048 89.048-89.048s89.047 39.86 89.047 89.048-39.86 89.047-89.047 89.047zm0-135.764c-25.79 0-46.677 20.887-46.677 46.677 0 25.789 20.887 46.676 46.677 46.676 25.789 0 46.676-20.887 46.676-46.676-.001-25.75-20.887-46.677-46.676-46.677zm-216.521 46.717c0 25.789-20.887 46.676-46.676 46.676s-46.676-20.887-46.676-46.676c0-25.79 20.887-46.677 46.676-46.677 25.789-.04 46.676 20.887 46.676 46.677z"/></svg>
```

Reply:
Whoops, sorry about that! Thanks for fixing that!